packages/fftw3/fftw3.0.7.2/opam

```diff
@@ -15,7 +15,19 @@ build: [
       "--localstatedir=%{lib}%/fftw3/var"
       "--libdir=%{lib}%/fftw3/lib"
       "--includedir=%{lib}%/fftw3/include"
-      "--datarootdir=%{lib}%/fftw3/share"]
+      "--datarootdir=%{lib}%/fftw3/share"] { os != "darwin" }
+  ["./configure" 
+      "CFLAGS=-I/usr/local/include"
+      "LDFLAGS=-L/usr/local/lib"
```

Comment:
If I understand well, the default location to install FFTW3 on MacOS — including through a package manager — is `/usr/local/`?